client/src/myLinksPage/Table.svelte

```diff
@@ -319,6 +320,13 @@ function copyToClipboard(text) {
     dummy.select();
     document.execCommand("copy");
     document.body.removeChild(dummy);
+    toast.push('Link Copied',{
```

Comment:
Instead of having this object at 3 places, put them in constants and export them